infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackService.kt

```diff
@@ -167,6 +167,11 @@ class LocationTrackService(
         return deletedVersion
     }
 
+    @Transactional
+    fun fetchDuplicates(id: IntId<LocationTrack>) = dao
```

Comment:
Tässä olisi syytä antaa tuo publishtype parametrina sisään, ihan konvention vuoksi - kun näin tuon funkkarin käyttöpaikan ensin, ihmettelin että mitäs versioita tuon on tarkoitus hakea, vai hakeeko jostain syystä molempia.